src/components/ExerciseItem/index.tsx

```diff
@@ -13,31 +19,48 @@ import {
   ExerciseDetailImage,
   ExtandButton,
   ButtonsContainer,
-  ButtonLink,
   DetailsList,
   DetailItem,
 } from "./ExerciseItem.styles";
-
-import { useUserContext } from "../../contexts/userContext";
-
-import { PATHS } from "../../pages/paths";
 import { Exercise } from "../../shared/interfaces";
+import { AddWorkoutForm } from "../../hooks/formHooks/workout/useNewWorkoutForm";
+import { defaultSets } from "../../hooks/formHooks/workout/constans";
 import { Role } from "../../shared/enums";
 
+import { ExerciseProgressModal } from "../../modals";
+
 type ExerciseItemProps = {
   exercise: Exercise;
+  appendExercise: UseFieldArrayAppend<AddWorkoutForm, "exercises">;
```

Comment:
exercises to powinien być enum, staraj się nie wpisywać takich wartości z palca bo łatwo o błąd i nie mamy kontroli nad typowaniem